pkg/commands/workload.go

```diff
@@ -616,21 +618,56 @@ func (opts *WorkloadOptions) Create(ctx context.Context, c *cli.Config, workload
 func (opts *WorkloadOptions) LoadInputWorkload(input io.Reader, workload *cartov1alpha1.Workload) error {
 	var in io.Reader
 
-	f, err := os.Open(opts.FilePath)
-	in = f
-	if f == nil && opts.FilePath == "-" {
+	isURL, err := isUrl(opts.FilePath)
+	if err != nil {
+		return fmt.Errorf("unable to check if filepath %q is a valid url: %w", opts.FilePath, err)
+	}
+	switch {
+	case isURL:
+		in, err = opts.getUrlFileContent()
+		if err != nil {
+			return fmt.Errorf("unable to read file from url %q: %w", opts.FilePath, err)
```

Comment:
please @danfein check this message, i think using _resource_ instead of _file_ is better suited

Reply:
Talked with @warango4  - suggested simplifying down to `unable to read from url <value>` 

Shortens the message, and sidesteps the question of what to call the thing we are reading. :)